### ISSUE TITLE
[ERROR] Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by jay009id** [jhunt/pgrouter#5]
Halo, I got the same error as #3. What is the authdb telling about? How to generate that file? How to set the IP (host) for write and the other is for select query?

**Comment by jhunt** [comment]
Is this an environment you can run something like valgrind against?

**Comment by jay009id** [comment]
No, I'm running it in the docker container.

**Comment by jhunt** [comment]
Can you share more details about the Dockerized postgres setup?  I'd like to be able to reproduce this locally.

**Comment by jhunt** [comment]
I was able to repro this by using a configuration that didn't explicitly define the user to run as.  Is that true about your config, @jay009id ?

**Comment by jay009id** [comment]
Correct, I don't add authdb in the config. I don't know how to generate auth file such as password.sample. Can you guide me how to do it?
